subworkflows/local/align.nf

```diff
@@ -0,0 +1,50 @@
+//
+// Map to reference
+//
+
+include { ALIGN_BWAMEM2  } from './alignment/align_bwamem2'
+include { ALIGN_SENTIEON } from './alignment/align_sentieon'
+
+workflow ALIGN {
+    take:
+        ch_reads_input     // channel: [mandatory] [ val(meta), [path(reads)]  ]
+        ch_fasta           // channel: [mandatory] [ path(fasta) ]
+        ch_fai             // channel: [mandatory] [ path(fai) ]
+        ch_index_bwa       // channel: [mandatory] [ val(meta), path(index) ]
+        ch_index_bwamem2   // channel: [mandatory] [ val(meta), path(index) ]
+        ch_known_dbsnp     // channel: [optional; used by sentieon] [ path(known_dbsnp) ]
+        ch_known_dbsnp_tbi // channel: [optional; used by sentieon] [ path(known_dbsnp_tbi) ]
+        val_platform       // string:  [mandatory] illumina or a different technology
+
+    main:
+        ch_versions   = Channel.empty()
+
+        ALIGN_BWAMEM2 (
+            ch_reads_input,
+            ch_index_bwamem2,
+            ch_fasta,
+            ch_fai,
+            val_platform
+        )
+
+        ALIGN_SENTIEON (
+            ch_reads_input,
+            ch_fasta,
+            ch_fai,
+            ch_index_bwa,
+            ch_known_dbsnp,
+            ch_known_dbsnp_tbi,
+            val_platform
+        )
```

Comment:
I feel like ALIGN_* are actually doing a bit more than just ALIGN, but ok enough.
Just wondering how do you control if it's BWAMEM2 or SENTIEON as it's not obvious there is the code.
Some extra comments would be helpful for contributors.

Reply:
Addressed here https://github.com/nf-core/raredisease/pull/348/files